server/src/main/java/org/opensearch/threadpool/ThreadPoolStats.java

```diff
@@ -197,6 +215,7 @@ static final class Fields {
         static final String REJECTED = "rejected";
         static final String LARGEST = "largest";
         static final String COMPLETED = "completed";
+        static final String WAIT_TIME = "thread_pool_wait_time";
```

Comment:
should we rename it to `wait_time_sum` given it is part of `thread_pool` stats so we don't need that as an explicit prefix.

Reply:
I was thinking about this too actually -- do you think `total_wait_time` is better or just `wait_time`?

Reply:
`total_wait_time` sounds good to me.